contracts/OlasMech.sol

```diff
@@ -14,7 +14,7 @@ abstract contract OlasMech is Mech, IErrorsMech, ImmutableStorage {
     event Request(address indexed sender, uint256 requestId, bytes data);
     event RevokeRequest(address indexed sender, uint256 requestId);
 
-    enum PaymentType {
+    enum PaymentType { // this level of the contract should not have to specify its possible extensions. So Payment type needs to be something that's extensible, string or number..
```

Comment:
@kupermind please fix!

Reply:
Agree, I didn't like this hardcoding either

Reply:
hash is pretty, as always :)